src/dra/6D59C.c

```diff
@@ -71,7 +71,68 @@ void func_8010D59C(void) {
     }
 }
 
-INCLUDE_ASM("dra/nonmatchings/6D59C", func_8010D800);
+u8 g_D_800ACF2C[] = {4, 4, 4, 4, 6, 6, 6, 6, 8, 8, 16, 16, 16, 16, 16, 16};
```

Comment:
If this is all 16s the shadows fade out very quick. `g_shadowOpacityReductionTable` or something?